keyboards/noll/multisteno/keyboard.c

```diff
@@ -0,0 +1,54 @@
+/* Copyright 2022 Nathan Olivares
+ * 
+ * This program is free software: you can redistribute it and/or modify 
+ * it under the terms of the GNU General Public License as published by 
+ * the Free Software Foundation, either version 2 of the License, or 
+ * (at your option) any later version. 
+ * 
+ * This program is distributed in the hope that it will be useful, 
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of 
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the 
+ * GNU General Public License for more details. 
+ * 
+ * You should have received a copy of the GNU General Public License 
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>. 
+ */
+#include <ch.h>
+#include <hal.h>
+#include "util.h"
+#include "quantum.h"
+
+#ifdef BOARD_STM32_F103_STM32DUINO
+
```

Comment:
There's probably a better way to do all this. What exactly are you trying to achieve here?